### ISSUE TITLE
Predicting same label every time

### ISSUE THREAD

**Issue by sakshijain032** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#474]
I have 2 classes in training data (Gun, Knife)but when I test on Gun image I am getting a knife as it's label.I have made changes in generate_tfrecored and object_detection.mbtxt but still facing the same problem
